src/cms/content-staging-preview.md

```diff
@@ -6,46 +6,50 @@ title: Previewing a Campaign
 Any day or period of time during a campaign can be previewed and shared with others.
 
 ![]({% link images/images-ee/content-staging-preview.png %}){: .zoom}
-_Preview_
+_Preview page controls_
 
-## To preview a campaign:
+1. Use one of these methods to open a Preview window:
```

Comment:
should Preview be ital?

Reply:
I am actually going to change to lowercase. This is not really a proper name. Similar to browser page or application window. I'm going to update that in other places also.